phpmyfaq/src/phpMyFAQ/Controller/Administration/NewsController.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+/**
+ * The Admin News Controller
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * @package   phpMyFAQ
+ * @author    Thorsten Rinne <thorsten@phpmyfaq.de>
+ * @author    Jan Harms <modelrailroader@gmx-topmail.de>
+ * @copyright 2024 phpMyFAQ Team
+ * @license   https://www.mozilla.org/MPL/2.0/ Mozilla Public License Version 2.0
+ * @link      https://www.phpmyfaq.de
+ * @since     2024-04-20
+ */
+
+namespace phpMyFAQ\Controller\Administration;
+
+use DateTime;
+use phpMyFAQ\Configuration;
+use phpMyFAQ\Controller\AbstractController;
+use phpMyFAQ\Entity\NewsMessage;
+use phpMyFAQ\Enums\PermissionType;
+use phpMyFAQ\Filter;
+use phpMyFAQ\News;
+use phpMyFAQ\Session\Token;
+use phpMyFAQ\Translation;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class NewsController extends AbstractController
+{
+    #[Route('admin/api/news/add')]
+    public function addNews(Request $request)
+    {
+        $this->userHasPermission(PermissionType::NEWS_ADD);
+        $data = json_decode($request->getContent());
+
+        $faqConfig = Configuration::getConfigurationInstance();
```

Comment:
You don't need this, you can get the configuration instance from

`$this->configuration`

as it's defined and instantiated in the AbstractController